### PR TITLE
table missing controls #3832

### PR DESCRIPTION
**Pull request by mnavasiolava**
the limit of items to show pagination controls was changed to 15. and they are shown without a scrollbar

### CODE REVIEW

**Comment by tomca32** [comment]
Could we maybe put these values in globals? As they stand now they are pretty much magic numbers and it's not obvious why they have those values.

**Comment by scottdraves** [comment]
disagree that they should be globals since they are only used in this one place.
but a comment that describes their relationship would be good.
plus, does doesn't this assume 30 pixels per row?  in advanced mode that's not true anymore...

**Comment by tomca32** [comment]
Fair point about globals. How about constants defined at the beginning of the file? Or a comment describing the relationship, I like to avoid comments if we can simply name the variable something like 
MIN_ITEMS_FOR_PAGINATION

**Comment by mnavasiolava** [comment]
added constants for number of rows for paging and row height
also took into account different row heights in normal and advanced modes

**Comment by scottdraves** [comment]
kick jenkins

File: core/src/main/web/outputdisplay/bko-tabledisplay/bko-tabledisplay.js [hunk 1]
```diff
@@ -1589,20 +1592,20 @@
           };
 
           var domCommon = '<"bko-table"Z' + (scope.data.length > 500 ? 'r' : '') + 't';
+          var rowHeight = bkHelper.getBkNotebookViewModel().isAdvancedMode() ? ROW_HEIGHT_ADVANCED_MODE : ROW_HEIGHT;
           if (!scope.pagination.use) {
             init.paging = false;
-            init.scrollY = scope.pagination.rowsToDisplay * 27 + 2;
+            init.scrollY = scope.pagination.rowsToDisplay * rowHeight + 2;
```
Comment (tomca32) [inline core/src/main/web/outputdisplay/bko-tabledisplay/bko-tabledisplay.js:1598]: did you mean ROW_HEIGHT here?

**Comment by scottdraves** [comment]
i am seeing the controls even though there are only 23 rows, and one page.

File: core/src/main/web/outputdisplay/bko-tabledisplay/bko-tabledisplay.js (hunk 1 above)
Comment (mnavasiolava) [inline core/src/main/web/outputdisplay/bko-tabledisplay/bko-tabledisplay.js:1598]: no, i meant rowHeight which is different for advanced and normal mode

Comment (tomca32) [inline core/src/main/web/outputdisplay/bko-tabledisplay/bko-tabledisplay.js:1598]: Oh, I see. Cool, thx

**Comment by mnavasiolava** [comment]
@scottdraves but you asked to change the limit of rows to show pagination controls to 15 or there is smth i misunderstood?

**Comment by scottdraves** [comment]
i meant it should do the obvious thing:
if there is 1 page then do not show pagination controls.
if there is > 1 page then do show.

**Pull request merged**